examples/create-inner-asset/assets/worker_app.py

```diff
@@ -0,0 +1,35 @@
+from pyteal import *
```

Comment:
1. please add a doc comment to what this app is doing, also say that his is demo app and doesn't have security measures. Make similar description in the README file.
2. maybe let's rename to something more meaningful? How about "logger.py"?
3. Also, I think we can simplify and remove not used code (eg delete check ...)

Reply:
Done.